README.md

```diff
@@ -39,4 +39,5 @@ The notification module will process the notification and update the matching co
 
 - Follow [Integration Guide](./notification/docs/IntegrationGuide.md) for information how to integrate with notification module.
 - Follow [Deployment Guide](./notification/docs/DeploymentGuide.md) to run notification module.
+- Follow [Continuous Deployment Guide](docs/CONTINUOUS_DEPLOYMENT_GUIDE.md) to configure deployment pipelines for both modules.
```

Comment:
It looks like it's only documented for the notification module, maybe we need to add this into a new section.

Reply:
15d435f1bec9b1eab0407fcaa532007f89dc016b